src/java.security.jgss/windows/native/libsspi_bridge/sspi.cpp

```diff
@@ -27,7 +27,7 @@
 // It speaks krb5 and SPNEGO. NTLM is excluded from SPNEGO negotiation.
 //
 // This library can be built directly with the following command:
-//   cl -I %OPENJDK%\src\java.security.jgss\share\native\libj2gss\ sspi.cpp \
+//   cl -I %OPENJDK%\src\java.security.jgss\share\native\libj2gss\ sspi.cpp
 //      -link -dll -out:sspi_bridge.dll
```

Comment:
I think the backslash `\` should remain there: it's a command line and `\` is used to wrap the long line.

Reply:
I did think about that too when I first saw it, but the command this is often run from is the Windows CMD, which does not accept `\` to escape to the next line. MSYS bash does not have the required environment set up outside of configure time to run `cl` either, but I have not tested with WSL yet. I suppose one could do the complicated setup required to be able to run this from the winenv provided bash, but that seems like a lot of hassle, enough that to me someone trying to compile sspi_bridge.dll by itself wouldn't really bother with the process. On top of that cl.exe will actually interpret the backslash as the literal object file `\.o` for some weird reason. I've already pinged the original author of the file @wangweij above to ask for guidance on this as such

Reply:
I used to have Visual Studio compilers available in my Cygwin environment. Now I don't, I just point the `configure` script to the batch file which configures the environment.

If this command is supposed to be run in `cmd.exe`, then `^` makes more sense: one could just copy & paste the command.

Reply:
I've just tried copying and pasting the command with ^ instead of \ and cmd doesn't seem to be correctly reading the compile command either even with ^, is there something I'm missing here?

Reply:
Honestly, with all of this confusion around the command line, I think I'd rather just avoid the issue entirely by turning this into a multiline comment so the backslash doesn't cause compiler warnings

Reply:
I don't know what the error message you get, but `^` definitely works for me as continuation line marker in `cmd.exe`.

```txt
C:\dev\jdk-dev>cl -I %OPENJDK%\src\java.security.jgss\share\native\libj2gss\ sspi.cpp ^
More?    -link -dll -out:sspi_bridge.dll
'cl' is not recognized as an internal or external command,
operable program or batch file.
```

This `More?` prompt identifies the continuation line.

Anyway, we decided to leave it as is but convert to multi-line comment, which is fine with me.